### PR TITLE
fix(client): add relevant placeholder to link submission

### LINKED ISSUE #38368: Unclear instructions with the use of Glitch to submit challenges.
I was recently attempting to submit my code changes of the [APIs and Mcroservices certification ](https://www.freecodecamp.org/learn/apis-and-microservices/managing-packages-with-npm/) through the URL of glitch as instructed in the introduction but I was finding trouble with passing the tests because I was submitting a wrong URL. Instead of something like  `https://blush-tundra-dodo.glitch.me/` I was submitting  `https://glitch.com/edit/#!/blush-tundra-dodo?path=package.json:7:21` until I was guided on the help forum before I got demotivated.

Don't you think we should revise the introduction and include steps on how to submit in the URLs from glitch to save future cases like mine? How do you see?

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #38368

This pull request is to correct the rebase issue with my previous pull request.

### CODE REVIEW

**Review by ojeytonwilliams — COMMENTED** [review]
This works perfectly, thanks @Sky020!  

We could refactor this further by renaming `ProjectForm` to `SolutionForm`, since it's no longer exclusively used for projects.  Also, `isBackEndProject` could become `isProject`, though the logic would need slightly tweaking in that case, since the frontend projects should be passing `true` for `isProject`.

I appreciate this is a bit nitpicky, though, so (if you think it's reasonable) I'd be happy to add a commit with these changes.

File: client/src/templates/Challenges/projects/backend/Show.js [hunk 1]
```diff
@@ -175,15 +161,12 @@ export class BackEnd extends Component {
         challengeMeta: { introPath, nextChallengePath, prevChallengePath }
       },
       tests,
-      isSignedIn,
       executeChallenge,
-      updateProjectFormValues
+      updateSolutionFormValues
```
Comment (ojeytonwilliams) [inline client/src/templates/Challenges/projects/backend/Show.js]:
```suggestion
      updateSolutionFormValues,
      updateBackendFormValues
```
This is needed for challenges.

File: client/src/templates/Challenges/projects/backend/Show.js [hunk 2]
```diff
@@ -203,21 +186,12 @@ export class BackEnd extends Component {
...
-                    isFrontEnd={false}
-                    onSubmit={executeChallenge}
-                    updateProjectForm={updateProjectFormValues}
-                  />
-                )}
+                <SolutionForm
+                  isFrontEnd={false}
+                  isProject={isBackEndProject}
+                  onSubmit={executeChallenge}
+                  updateSolutionForm={updateSolutionFormValues}
```
Comment (ojeytonwilliams) [inline client/src/templates/Challenges/projects/backend/Show.js]:
```suggestion
                  updateSolutionForm={values =>
                    isBackEndProject
                      ? updateSolutionFormValues(values)
                      : updateBackendFormValues(values)
                  }
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Thanks for going through and refactoring those files.  I noticed a small bug with form updates and I've added suggestions to fix that.

Unfortunately there are a couple of remaining references to `handleSubmit` in `backend/Show.js` that I can't add suggestions to.  Would you mind removing those, since that's all dealt with in `SolutionForm` now.

**Comment by Sky020** [comment]
> Thanks for going through and refactoring those files. I noticed a small bug with form updates and I've added suggestions to fix that.
> 
> Unfortunately there are a couple of remaining references to `handleSubmit` in `backend/Show.js` that I can't add suggestions to. Would you mind removing those, since that's all dealt with in `SolutionForm` now.

@ojeytonwilliams , I changed them in a new commit, as I was not sure what would happen if I accepted your suggestions for commit. _Git_ and I seldom get along.

**Review by ojeytonwilliams — APPROVED** [review]
Looks good to me!

**Comment by ojeytonwilliams** [comment]
>  I changed them in a new commit, as I was not sure what would happen if I accepted your suggestions for commit.

It creates a new commit with just those changes.  It's remarkably painless.

> Git and I seldom get along.

Don't worry about it, either way works.  Git is legendarily difficult to get started with, so you're not the only one it disagrees with.

Note for anyone QAing: ideally check submissions for the affected challenges still behave themselves.  It *should* be fine, but a second look wouldn't hurt.

**Pull request merged**